test/unit/mongo/MongoTripodDriverTest.php

```diff
@@ -91,9 +91,10 @@ public function testSelectSingleValue()
         $this->assertEquals($expectedResult,$actualResult);
     }
 
-    public function testDescribe()
+    public function testGraph()
     {
-        $expectedResult =
+        $expectedResult = new \Tripod\ExtendedGraph();
+        $expectedResult->add_turtle(
             "<http://talisaspire.com/resources/3SplCtWGPqEyXcDiyhHQpA> <http://purl.org/dc/terms/isVersionOf> <http://talisaspire.com/works/4d101f63c10a6> .
```

Comment:
Eventually we should probably move these nearly 90 lines of ntriples (!!) into their own fixture files, but not on this day.